_posts/2017-10-05-apprenticeship-at-8thlight.md

```diff
@@ -0,0 +1,213 @@
+Mentoring needs some experience too, the mentor Daniel Irvine writes in [his blog][4-tips-for-mentoring] “go too fast and they won't take everything in; go too slow and they'll be bored”. Mentors must not be too specific, leave room for learning and find the right dose of challenge.
+“Mentors are training their future team members, so they are personally invested in the success of each apprentice” is written [on the 8th Light apprenticeship page][8thlight-apprenticeship].
+
+The time a mentor invests is about half a day a week. There are mentors who take more time to work with their apprentices, but there is no fix rule. 
+
+[8thlight-apprenticeship]: https://8thlight.com/apprenticeship/
+[4-tips-for-mentoring]: https://dev.to/d_ir/4-tips-for-mentoring-developers-894
+
+## Regular Meetings
+
+As mentioned above there are regular meetings between apprentice, mentor and co-mentor. The one and most important meeting is the IPM, the iteration planning meeting. It takes place once a week for about one hour. During this meeting the apprentice reports the progress of the last iteration and asks questions that might come up. The mentors will help answering those questions and might adjust the plan for the upcoming iterations. Later the next iteration is planned. The tasks are moved from the backlog into the to-do column and the apprentice estimates them. The mentors support doing this as well as possible.
+
+All the mentors meet once a month for the MMM, the monthly mentor meeting. In this meeting the mentors exchange ideas on the mentoring and also on the apprentices.
+Especially when in doubt about an apprentice’s progress this meeting serves to support each other.
+
+The review board is another meeting where a selected group of people comes together for reviewing a certain apprentice. This board meets at the end of the apprenticeship to evaluate the apprentice. If there is a veto option by this board or not, I don’t know. The important thing I think is that this board comes together to evaluate a certain apprentice. If this takes place more often I don’t remember.
+
+Because the apprentices have an alike experiences they meet for an apprentice learning group. This group is apprentices only, to learn from each other, exchange ideas and just chat. Apprentices see the things different than their mentors and other crafters, they need the room for their own exchange.
+
+## Graduation
+
+Before graduating to become a crafter every apprentice has to go for a challenge. This is a project of its own, that the apprentice has to do from start to finish, ideally in those two weeks.
+And the apprentice will join some other crafter of the company to follow them around and [shadow][shadowing] them for a week.
```

Comment:
We call the shadowing a "pairing tour". The intent is for the apprentice to pair for at least one day with each of the 8 members of their review board.